root/static/scripts/edit/URLCleanup.js

```diff
@@ -1816,8 +1816,13 @@ const CLEANUPS = {
     },
   },
   'secondhandsongs': {
-    match: [new RegExp('^(https?://)?([^/]+\\.)?secondhandsongs\\.com/', 'i')],
+    match: [new RegExp('^(https?://)?(?:[^/]+\\.)?secondhandsongs\\.com/', 'i')],
```

Comment:
If you go this direction, none of the groups in `match` rules needs to be capturing, but that should probably be done in a separate commit.